src/ec/packages/ec-component-button/button.story.js

```diff
@@ -27,147 +28,100 @@ uiIcons.forEach(icon => {
   iconsList[icon] = icon;
 });
 
+const prepareButton = data => {
```

Comment:
Well, instead of repeating this exception you can set it at the top of the file like this:
`/* eslint-disable no-param-reassign */`